templates/prometheus-exporter/README.md

```diff
@@ -17,3 +17,5 @@ Add thresholds when appropriate.
 
 Import the .cfg file to add it to list of Templates.
 Set the %{ENDPOINT} and %{PORT} macros with the appropriate values.
+
+For more information about the Prometheus Exporter Template, refer to [Consuming Prometheus Metrics in Monitoring Studio X](https://www.sentrysoftware.com/support/user-documentation.html).
```

Comment:
Please link to Monitoring Studio X's documentation: https://www.sentrysoftware.com/library/swsyx/index.html

Reply:
I added the link that @iguitton suggested.

Reply:
Please update the link (you can't label a link "Consuming Prometheus Metrics in Monitoring Studio X" and make it go to Sentry's list of all product docs).

Reply:
@bertysentry , the link https://www.sentrysoftware.com/library/swsyx/prometheus/consuming-prometheus-metrics-in-patrol.html will only be effective when the Monitoring Studio 10.3 documentation is published on Sentry Website. In the meanwhile, you will be redirected to the page listing all Sentry User Docs.